notebooks/output-sanitize.cfg

```diff
@@ -10,22 +10,22 @@ replace: /STATUS_FILE
 regex: \d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}Z
 replace: DATE_TIME
 
-[regex3]
+[regex4]
+regex: \d{4}-\d{2}-\d{2}
```

Comment:
I had the date surrouned by '/' might want to keep it and maybe ' ' space or other delimiter chars?  Just try to prevent matching too much and miss a real error.